pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
> when they try to create such an object

Is [route status validation](https://github.com/openshift/origin/blob/3cf7f3cccd5b7b95448568ba8e2b3c81cc6ab9af/pkg/route/registry/route/strategy.go#L99-L101) running when creating a route or only when a route is updated?